RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -137,6 +137,68 @@ private static Map<String, String> overrideAndRemoveEnv(
         "docker.elastic.co/elasticsearch/elasticsearch:5.5.2",
         Version.fromString("ES 5.5.2")
     );
+    public static final ContainerVersion ES_V5_4 = new ElasticsearchVersion(
+            "docker.elastic.co/elasticsearch/elasticsearch:5.4.2",
+            Version.fromString("ES 5.4.2")
+    );
+    public static final ContainerVersion ES_V5_3 = new Elasticsearch5Version(
+        "elasticsearch:5.3.2",
+        Version.fromString("ES 5.3.2"),
+        "/usr/share/elasticsearch/config/elasticsearch.yml",
+        String.join("\n",
```

Comment:
Please refactor these duplicate settings following DRY principles https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Seems like there is alot of unneeded / redundant details in this file.  Please do a pass over each part to see if it is needed - less lines of code is easier to maintain and improve on in the future.